stdlib/array.gr

```diff
@@ -681,14 +670,10 @@ export let count = (fn, array) => {
  */
 export let counti = (fn, array) => {
   let length = length(array)
-  let mut position = 0
   let mut count = 0
-  while (position < length) {
-    if(fn(array[position], position)) {
+  for (let mut position = 0; position < length; position = incr(position)) {
```

Comment:
```suggestion
  for (let mut position = 0; position < length; position += 1) {
```

Reply:
Replaced all remaining `incr` calls in the file